app/controllers/items_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ItemsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
```

Comment:
actually it needs to be `current_user`

On Fri, Jan 8, 2016 at 1:51 PM, Aaron Kozminski <aaronkoz74@gmail.com>
wrote:

> I need to add `authenticate_user!
>
> On Fri, Jan 8, 2016 at 1:01 PM, bjblock <notifications@github.com> wrote:
>
>> In app/controllers/items_controller.rb
>> <https://github.com/aaronkoz74/blocitoff/pull/5#discussion_r49217131>:
>>
>> > @@ -0,0 +1,19 @@
>> > +class ItemsController < ApplicationController
>> > +  def new
>> > +    @user = User.find(params[:user_id])
>>
>> does this mean I can go to any user's page and create items for that user?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/aaronkoz74/blocitoff/pull/5/files#r49217131>.
>>
>
>